Networking/Networking/Model/Blaze/BlazeCampaign.swift

```diff
@@ -70,7 +70,9 @@ public struct BlazeCampaign: Decodable, Equatable, GeneratedFakeable, GeneratedC
         name = try container.decode(String.self, forKey: .name)
         uiStatus = try container.decode(String.self, forKey: .uiStatus)
         budgetCents = try container.decode(Double.self, forKey: .budgetCents)
-        productURL = try container.decode(String.self, forKey: .targetUrl)
+
+        let targetUrn = try container.decode(String.self, forKey: .targetUrn)
+        productID = Int64(String(targetUrn.split(separator: ":").last ?? ""))
```

Comment:
nit: It might help to add a comment mentioning the expected format.

Reply:
Added comment in da40b80cc49ef99ab827f770b58c6acccfc9685b.